scripts/loadable_app/hello.c

```diff
@@ -0,0 +1,12 @@
+int a = 9;
+int b = 9;
+
+int x()
+{
+return 7;
+}
+int main(int argc, char **argv)
+{
+	int p = x();
+	return a + p + x();
```

Comment:
ok, how to use these toolchains? i've tried ` export PATH=$PATH:i386-elf-toolchain` and the `./i386-elf-gcc-6.3.0 hello.c`, but got an error:  ` cannot find crt0.o: No such file or directory`